app/helpers/multi_languages_helper.rb

```diff
@@ -99,8 +99,10 @@ def content_language_selector(id: 'content_language', name: 'content_language')
 
 
   def language_hash(concept_label)
-
-    return concept_label.first if concept_label.is_a?(Array)
+    if concept_label.is_a?(Array)
```

Comment:
this is not correct for `prefLabels`, which needs to return only one even there is multiple prefLabels

Reply:
Is this related only to prefLabels only?

Reply:
not really, this function `language_hash`has for goal to get you the correct value for a correspondent language. is you choose `all` it returns an hash of the values in all the languages

Reply:
It returns always a single value !

Reply:
Try to show all languages with AGROVOC.

Reply:
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/18340b19-fbaf-411e-9ea5-dad195395a60)